docs/ske/03-integrations/01-tfe.mdx

```diff
@@ -0,0 +1,425 @@
+name=$(yq '.metadata.name' /kratix/input/object.yaml)
+name=$(yq '.metadata.namespace' /kratix/input/object.yaml)
+cat <<EOF > /kratix/output/${name}-${namespace}.tf
+output "${name}-${namespace}" {
+  value = "some-value"
+}
+EOF
+```
+
+Since the Promise workflow will run once per resource, the outputs it generates
+need to be unique. The example above uses the resource's name and namespace to
+generate a unique filename and output name. You may need to adapt your code to follow the
+same pattern.
+
+## Adding the SKE TFE Output Aspect
+
+If you followed the steps above, you should now have a Promise that schedules
+workloads to TFE. However, given that the Terraform code will be planned and
+executed by TFE, the users of your Platform won't have access to the Terraform
+outputs directly.
+
+To solve this, you can use the SKE TFE Output Aspect. This Aspect will pull the
+Terraform outputs from TFE and make them available in the Resource's status.
+
+To use the Aspect, you need to add it to your Promise as a second Pipeline in
+the `spec.resource.configure` workflow:
+
+```yaml
+apiVersion: platform.kratix.io/v1alpha1
+kind: Promise
+# ...
+spec:
+  workflows:
+    resource:
+      configure:
+        - apiVersion: platform.kratix.io/v1alpha1
+          kind: Pipeline
+          # ... workflow that generates the Terraform code
+
+        - apiVersion: platform.kratix.io/v1alpha1
+          kind: Pipeline
+          metadata:
+            name: output-writer
+            namespace: default
+          spec:
+            containers:
+              - image: ghcr.io/syntasso/ske-tfe-output:v0.1.0
+                name: fetch-output
+                env:
+                - name: TIMEOUT
+                  value: "30m"
+```
+
+The `TIMEOUT` environment variable should be configured to match the amount of
+time you expect the Terraform plan and apply to take. The Aspect will keep
+trying to fetch the outputs until the timeout is reached.
+
+
+:::info
+
+The `ske-tfe-output` image is hosted in a private registry. To access the
+image, you will need to use the access token provided by Syntasso and your
+Kubernetes cluster must have the permissions to pull images from the registry.
+
+You can also set `imagePullSecrets` in the [Pipeline
+spec](/main/reference/workflows#pipelines).
+
+:::
+
+By default, the Aspect will add the Terraform outputs in the Resource's
+`status.outputs`. In the example Promise above, the following would be added to the
+resource status:
+
+```yaml
+outputs:
+  tfe: # Destination name
+    resource-configure: "some-value" # TFE output name/value
+```
+
+If you want to customise what gets persisted, you can set the `SKIP_STATUS` environment
+variable in the Aspect's container to `"true"`. When the flag is set, the Aspect will
+create a set of `*-output.yaml` files in `/kratix/metadata` containing the Terraform
+outputs. You can then use those files to make your own updates to the status in
+`/kratix/metadata/status.yaml`.
+
```

Comment:
shall we show a simple example of this in the example promise? IMO it will be really common

Reply:
Good shout!